_includes/map.html

```diff
@@ -0,0 +1,22 @@
+<div class='full' style='background: #F7F7F4; padding-top: 0px;'>
+  <div class="row">
+    <div class="large-12 columns">
+
+      <div class="mod modSectionHeader ">
+        <div class="special-title centered-text separateline">
+          <a name="alumni"></a>
+          <h2 class="separate" style="color:">Where we are from</h2>
+        </div>
+
+        <div style="height:420px; overflow:hidden;">
+          <iframe style="position:relative; top:-46px; border:none;" src="https://www.google.com/maps/d/embed?mid=1blontcPuFDDdssAujS91Y-w3d0eN486Z" width="1100" height="480"></iframe>
```

Comment:
This is not ideal. I assume this is a map in your account? That won't work in the long run. also, we're loosing any history changes. Let's use the google map javascript API for this. I can create an account for an API key tomorrow. Also, ask Taylor if he has any other suggestions.